parser/unparser_test.go

```diff
@@ -138,6 +140,191 @@ func TestUnparse(t *testing.T) {
+			in:   "a ? b : c ? d : e",
+			out:  "a ?\nb : (c ?\nd : e)",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Conditional),
+			},
+		},
+		{
+			name: "call_wrap_or",
+			in:   "a || b || c || d || e",
+			out:  "a ||\nb ||\nc ||\nd ||\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.LogicalOr),
+			},
+		},
+		{
+			name: "call_wrap_equals",
+			in:   "a == b == c == d == e",
+			out:  "a ==\nb ==\nc ==\nd ==\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Equals),
+			},
+		},
+		{
+			name: "call_wrap_greater",
+			in:   "a > b > c > d > e",
+			out:  "a >\nb >\nc >\nd >\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Greater),
+			},
+		},
+		{
+			name: "call_wrap_greater_equals",
+			in:   "a >= b >= c >= d >= e",
+			out:  "a >=\nb >=\nc >=\nd >=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.GreaterEquals),
+			},
+		},
+		{
+			name: "call_wrap_in",
+			in:   "a in b in c in d in e",
+			out:  "a in\nb in\nc in\nd in\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.In),
+			},
+		},
+		{
+			name: "call_wrap_less",
+			in:   "a < b < c < d < e",
+			out:  "a <\nb <\nc <\nd <\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Less),
+			},
+		},
+		{
+			name: "call_wrap_less_equals",
+			in:   "a <= b <= c <= d <= e",
+			out:  "a <=\nb <=\nc <=\nd <=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.LessEquals),
+			},
+		},
+		{
+			name: "call_wrap_not_equals",
+			in:   "a != b != c != d != e",
+			out:  "a !=\nb !=\nc !=\nd !=\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.NotEquals),
+			},
+		},
+		{
+			name: "call_wrap_divide",
+			in:   "a / b / c / d / e",
+			out:  "a /\nb /\nc /\nd /\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Divide),
+			},
+		},
+		{
+			name: "call_wrap_modulo",
+			in:   "a % b % c % d % e",
+			out:  "a %\nb %\nc %\nd %\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Modulo),
+			},
+		},
+		{
+			name: "call_wrap_multiply",
+			in:   "a * b * c * d * e",
+			out:  "a *\nb *\nc *\nd *\ne",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
+				WrapOnOperators(operators.Multiply),
+			},
+		},
+		{
+			name: "call_wrap_and_long_variables",
+			in:   "longVariableA && longVariableB && longVariableC",
+			out:  "longVariableA &&\nlongVariableB &&\nlongVariableC",
+			formattingOptions: []UnparserFormattingOption{
+				WrapColumn(3),
```

Comment:
Consider cases where the wrapColumn would not be applied, e.g. WrapColumn(10000) or WrapColumn(-1). Is there validation happening on the column value?

Reply:
The validation for invalid [column limit like -1 is done here](https://github.com/google/cel-go/pull/559/commits/604a4bd947dd3c724ca634f99d1e7a363fea1e74#diff-667600a8ae20f05557fb24c812a91e79c85833be4a084db2e1b05660d1eaf3c4R516-R520). The test triggers this [case at 0](https://github.com/google/cel-go/pull/559/commits/604a4bd947dd3c724ca634f99d1e7a363fea1e74#diff-2e0433ff11847f5508ec46127bc5ac5df75bd8001f0d324e67a01e722594c24eR445-R450). We could add another test for -1 for clarity if you'd like

Reply:
For `WrapColumn(10000)`, I thought about leaving it as is since this is technically a valid setting, albeit impractical. It might makes sense to set an upper limit that's reasonable for most screen sizes, say 1000.

Reply:
I mostly just thought it might be used to show that the line wrap doesn't happen at 1000 and also show that when the operator occurs _after_ the column specified how the code behaves.

Reply:
Sounds good, added a test case for this